src/Common/clickhouse_malloc.cpp

```diff
@@ -11,7 +11,7 @@ extern "C" void * clickhouse_malloc(size_t size)
     if (res)
     {
         AllocationTrace trace;
-        size_t actual_size = Memory::trackMemory(size, trace);
+        size_t actual_size = Memory::trackMemoryNoExcept(size, trace);
```

Comment:
This is now wrong since I changed the tracker to not account memory on OOM assuming callers would not call malloc(). Also, the way of handling it (via the trace) is terrible and I should do better. Good thing it's just an experiment